bin/lib/installable/archives.py

```diff
@@ -130,6 +136,23 @@ def verify(self) -> bool:
     def should_install(self) -> bool:
         return True
 
+    def save_version(self, exe: str, res_call: str):
+        if not running_on_admin_node:
+            return
+
+        relative_exe = "/".join(exe.split("/")[1:])
+        if self.install_path_symlink:
+            fullpath = self.install_context.destination / self.install_path_symlink / relative_exe
+        elif self.path_name_symlink:
+            fullpath = self.install_context.destination / self.path_name_symlink / relative_exe
+        else:
+            fullpath = self.install_context.destination / exe
+
+        stat = fullpath.lstat()
```

Comment:
Why `lstat` here? this seems to complicate things but I am probably missing something....

Naively I'd think a `stat` on the regular `Path` relative to the `destination` would be all you'd need?

Reply:
stat() seems to work